WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -781,25 +781,14 @@ private extension ProductFormViewController {
 
     /// Updates table viewmodel and datasource and attempts to animate cell deletion/insertion.
```

Comment:
Super nit: We should update the comment as we no longer animate the cell.

Reply:
Done in ef512e6e45cf54f913e7028b38426580385785af.